### PR TITLE
ER-540: Added docker support two nginx

### PR DESCRIPTION
**Pull request by cableman**

### CODE REVIEW

File: .docker/ereolen.vcl [hunk 1]
```diff
@@ -17,14 +17,43 @@ vcl 4.0;
 # Import VMod's
 # Used for std.healthy.
 import std;
+import directors;
 
 # Default backend definition. Set this to point to your content server.
-backend default {
+backend server1 {
```
Comment (rimi-itk) [inline .docker/ereolen.vcl]: Real computer scientists count from 0.

File: docker-compose.yml [hunk 2]
```diff
@@ -33,6 +33,15 @@ services:
       - ${PWD}/.docker/vhost.conf:/etc/nginx/conf.d/default.conf:ro
       - ./:/app:delegated
 
+  nginx2:
```
Comment (rimi-itk) [inline docker-compose.yml]: For consistency, consider calling the other `nginx` container `nginx1` (or `nginx0` if we count from 0).

File: .docker/ereolen.vcl (hunk 1 above)
Comment (cableman) [reply .docker/ereolen.vcl]: Fixed :-)

File: docker-compose.yml (hunk 2 above)
Comment (cableman) [reply docker-compose.yml]: Fixed :-)

**Pull request merged**